airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3769,6 +3769,9 @@ components:
             $ref: "#/components/schemas/JobConfigType"
         configId:
           type: string
+        includingJobId:
+          description: If the job with this ID exists for the specified connection, returns all jobs created after and including this job, or the full pagination pagesize if that list is smaller than a page. Returns an empty list if this job is specified and cannot be found in this connection.
```

Comment:
Should we rename it starting jobId? It sounds more descriptive than includingJobId

Reply:
Yeah I actually like startingJobId, I think that is better. I'll make that change 👍